programs/drift/src/ids.rs

```diff
@@ -98,5 +116,5 @@ pub mod usdt_oracle_mainnet {
 
 pub mod usdt_pull_oracle_mainnet {
```

Comment:
small nit: can we remove mainnet? should be the same for both for pull oracle

Reply:
yah